src/calibrate-multi-hop/app.py

```diff
@@ -0,0 +1,23 @@
+from controllers.calibrate import calibrate_bp
+from flask import Flask
+import logging
+import os
+from flask_cors import CORS
+from flask_pymongo import PyMongo
+from dotenv import load_dotenv
+load_dotenv()
+
+# import blue prints
+
+_logger = logging.getLogger(__name__)
+
+app = Flask(__name__)
+
+# Allow cross-brower resource sharing
+CORS(app)
+
+app.config["MONGO_URI"] = os.getenv("MONGO_URI")
```

Comment:
please share the .env for this micro in DM perhaps? Thanks. @bobturneruk

Reply:
The environment variables should be fully specified by the command line arguments in the readme (https://github.com/airqo-platform/AirQo-api/blob/Multi-hop/src/calibrate-multi-hop/readme.md#to-build-and-run-with-docker-desktop). Are there any we've missed, @Baalmart?